backend/routers/api/carousel.ts

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+import { GetCAROUSEL_DATA, SetCAROUSEL_DATA } from "../../constants/Carousels";
+
+const router = express.Router();
+
+// GET /carousel
+// Get carousel data
+router.get("/", (req, res) => {
+  res.status(200).json(GetCAROUSEL_DATA());
+});
+
+// POST /carousel
+// Post carousel data
+router.post("/", (req, res) => {
+  SetCAROUSEL_DATA(req.body);
+  res.status(200).json(GetCAROUSEL_DATA());
+});
```

Comment:
```suggestion
router.put("/", (req, res) => {
  SetCAROUSEL_DATA(req.body);
  res.status(200).json(GetCAROUSEL_DATA());
});
```

`HTTP PUT` is recommended to update data.